plutus-contract/src/Wallet/Emulator/Wallet.hs

```diff
@@ -70,13 +72,20 @@ walletAddress = pubKeyAddress . walletPubKey
 signWithWallet :: Wallet -> Tx -> Tx
 signWithWallet wlt = addSignature (walletPrivKey wlt)
 
-data WalletEvent = WalletMsg LogMessage
-    deriving stock (Show, Eq, Ord, Generic)
+data WalletEvent =
+    GenericLog T.Text
```

Comment:
This seems a little suspicious? Perhaps something we should try and get rid of later.

Reply:
https://jira.iohk.io/browse/SCP-933